Firmware/temperature.cpp

```diff
@@ -1023,21 +1023,9 @@ static void temp_runaway_stop(bool isPreheat, bool isBed)
 {
     if(IsStopped() == false) {
         if (isPreheat) {
-            lcd_setalertstatuspgm(isBed? PSTR("BED PREHEAT ERROR") : PSTR("PREHEAT ERROR"), LCD_STATUS_CRITICAL);
-            SERIAL_ERROR_START;
-            if (isBed) {
-                SERIAL_ERRORLNPGM(" THERMAL RUNAWAY (PREHEAT HEATBED)");
-            } else {
-                SERIAL_ERRORLNPGM(" THERMAL RUNAWAY (PREHEAT HOTEND)");
-            }
+            lcd_setalertstatuspgm(isBed? PSTR("THERMAL RUNAWAY (BED PREHEAT ERROR)") : PSTR("THERMAL RUNAWAY (PREHEAT HOTEND ERROR)"), LCD_STATUS_CRITICAL);
```

Comment:
Hmm I was perhaps a bit too agressive with optimisations here. This won't fit the LCD screen 🤔